packages/breadcrumbs/src/elements/Breadcrumb.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { Children, cloneElement, HTMLAttributes } from 'react';
+import { useBreadcrumb } from '@zendeskgarden/container-breadcrumb';
+
+import { StyledBreadcrumb, StyledBreadcrumbItem } from '../styled';
+
+export const Breadcrumb = React.forwardRef<HTMLElement, HTMLAttributes<HTMLElement>>(
```

Comment:
Since this component now forwards a ref, there should be a test for it, [similar to other packages](https://github.com/zendeskgarden/react-components/pull/529/files#diff-a658bb64292d200e01d13327f68d34d1R20).

Reply:
Test added 👍.